Orion Express Automated Station

DreamySkrell opened the pull request:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/efa02c06-cc66-4943-aa42-c0ea84963a56)

changes:
  - rscadd: "Adds the Orion Express Automated Station."

DreamySkrell commented:
!review

Greenjoe12345 commented on maps/away/away_site/orion_automated_station/orion_automated_station.dm:
```diff
@@ -0,0 +1,253 @@
+	)
+
+/obj/effect/overmap/visitable/sector/orion_automated_station/New()
+	designation = "[pick("Orion Express Automated Station #6282", "Orion Express Automated Station #1112", "Orion Express Automated Station #0091", "Orion Express Automated Station #9123", "Orion Express Automated Station #0157", "Orion Express Automated Station Zeta", "Orion Express Automated Station Kilo", "Orion Express Automated Station Uniform", "Orion Express Automated Station Whiskey")]"
```
Wouldn't it be better to have it as a thing of picking a number from 0001 to 9999 rather then these particular arbitrary numbers?

DreamySkrell replied:
done

Greenjoe12345 commented on maps/away/away_site/orion_automated_station/orion_automated_station.dm:
```diff
@@ -0,0 +1,253 @@
+	icon_state = "hallC"
+	area_blurb = "\
+	You hear a robotic voice coming from cheap ceiling-mounted speakers: \
+	'Welcome to the Orion Express Automated Station, where you may rest, exchange packages, and use any other of our facilities.' <br><br>\
```
I wonder if this part could be made to use a different kind of text span, as it is actively speaking

DreamySkrell replied:
I don not see a nice way to do this, as macros can't really be used in strings

Greenjoe12345 commented on maps/away/away_site/orion_automated_station/orion_automated_station.dm:
```diff
@@ -0,0 +1,253 @@
+	icon_state = "storage"
+	area_blurb = "\
+	You hear a robotic voice: \
+	'You may borrow any of the crates or canisters here, and return them later, free of charge. \
```
same here

DreamySkrell replied:
same

Greenjoe12345 commented on maps/away/away_site/orion_automated_station/orion_automated_station.dm:
```diff
@@ -0,0 +1,253 @@
+	icon_state = "storage"
+	area_blurb = "\
+	You hear a robotic voice: \
+	'You may leave your packages here, or take the packages to deliver elsewhere. \
```
ditto

DreamySkrell replied:
same

RustingWithYou commented on maps/away/away_site/orion_automated_station/orion_automated_station.dm:
```diff
@@ -0,0 +1,253 @@
+	spawn_weight = 1
+	suffixes = list("away_site/orion_automated_station/orion_automated_station.dmm")
+
+	sectors = list(ALL_POSSIBLE_SECTORS)
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```
```suggestion
	sectors = list(ALL_CORPORATE_SECTORS)
```

DreamySkrell commented:
this also needs entry points

DreamySkrell commented:
this should be nice and fixed and ready for review now

DreamySkrell commented:
!review

shaky-jake reviewed (changes requested):
Atmospherics, Packages Storage, Crates/Canisters Storage, and Engineering are all missing vents and scrubbers. I'd also give some longer hallway segments some as well. 

Additionally, the entire station is missing air alarms. Every segment that can be separated by emergency shutters should have one.

This is more optional, but I'd also change the `stand_icon = "banner"` for the flags on the walls. Looks a little out of place having the stand included on the wall.

Looks good besides that.

DreamySkrell commented:
> Atmospherics, Packages Storage, Crates/Canisters Storage, and Engineering are all missing vents and scrubbers. I'd also give some longer hallway segments some as well.

Added vents and scrubbers to both storage rooms. I think it's fine if not every room or hallway is covered by vents/scrubbers, especially so for smaller rooms or more "minimalist" stations like this one (and as long as there are adjacent rooms with vents/scrubbers).

> Additionally, the entire station is missing air alarms. Every segment that can be separated by emergency shutters should have one.

💀 I knew I forgot about something fundamental like this. Fixed now.

> This is more optional, but I'd also change the `stand_icon = "banner"` for the flags on the walls. Looks a little out of place having the stand included on the wall.

Done.

> Looks good besides that.

❤️

shaky-jake reviewed (approved):
Nice.
